packages/editor/src/components/browser/Browser.css

```diff
@@ -34,7 +34,12 @@
 }
 
 .insert {
-  color: var(--A400);
+  color: white;
```

Comment:
does this work in light and dark mode?

Reply:
Yes, i think so:
![image](https://github.com/axonivy/inscription-client/assets/141223521/5926ca94-ea9e-4b5b-802f-25403af94ec7)
![image](https://github.com/axonivy/inscription-client/assets/141223521/30380965-5961-47db-83ec-bd057f2b663b)